src/cli/cli.hpp

```diff
@@ -354,6 +354,23 @@ class Interpreter
     void ProcessLeaderWeight(uint8_t aArgsLength, char *aArgs[]);
 #endif
     void ProcessMasterKey(uint8_t aArgsLength, char *aArgs[]);
+#if OPENTHREAD_CONFIG_TMF_PROXY_MLR_ENABLE
+    void ProcessMlr(uint8_t aArgsLength, char *aArgs[]);
+
+#if OPENTHREAD_CONFIG_COMMISSIONER_ENABLE
```

Comment:
Yes, `MLR.req` is a message in scope of `MGMT`. So I think the command should be defined under `commissioner_cli.hpp`, right?

Reply:
I think you meant `cli_commissioner.hpp`. 
`cli_commissioner.hpp` only contains definitions for `commissioner ...` command. But `mlr ...` command will very likely be extended to have other commands that are not only for commissioners. So I would prefer to have `mlr` command defined in the `cli.hpp`.